projects/js-packages/publicize-components/src/hooks/use-attached-media/index.js

```diff
@@ -32,8 +37,21 @@ export default function useAttachedMedia() {
 		[ currentOptions, editPost ]
 	);
 
+	const updateIsSocialPost = useCallback(
+		option => {
+			editPost( {
+				meta: {
+					jetpack_social_options: { ...currentOptions, is_social_post: option },
```

Comment:
I wonder if it's worth it to combine this method and the `updateAttachedMedia` into a shared method that takes a key/value pair to update the relevant option.